src/lib/tutorials/normalize-product-like-slug.ts

```diff
@@ -21,7 +21,7 @@ export function normalizeSlugForDevDot(productSlug: string): ProductSlug {
 	} else if (isProductSlug(productSlug)) {
 		return productSlug
 	} else {
-		throw new Error(
+		console.warn(
```

Comment:
This was throwing during the tutorial map generation process since that deals with products that aren't yet in beta. I figure not throwing here might be better? Also could just be a temporary workaround until we find a better solution for the tutorial map generation.

Reply:
🚨 this is the patch for the issue on prod 🚨